2023/README.md

```diff
@@ -57,11 +57,9 @@ _________________
 * Add support for `Get` curl command for the test case.
 * Add support to `Run` individual test case.
 * On the CLI add a vertical line separator to show differences in the test cases 
-(show complete tests info on both sides, only differences in different color)
+(show complete tests info on both sides, only highlight differences with different color)
 * Add support for commands in Keploy CLI, for eg: `show [test_id];` will 
-show that specific test case among various of testruns, `show [test_run_id];` will show the info related to it.
-* Rename test files and mock files to method_endpoint.
```

Comment:
why is this and the following task removed? It's a GFI that can be done within this project scope.